couch/tests/test_couchv2.py

```diff
@@ -318,7 +318,7 @@ def test_compaction_metrics(aggregator, check, gauges, instance):
 
     update_url = '{}/{}'.format(url, body['_id'])
 
-    for _ in range(50):
+    for _ in range(100):
```

Comment:
Why does creating more test data will fix the flaky test ? Seems not a reliable way if the issue is about `_compact` being called too early.

Can we call an endpoint to verify that the data is created (before `_compact`), then continue ?

Reply:
`_compact` is not called too early, but finishes too quickly. The `_compact` needs to be running when calling the check in order to pass the tests. Adding more data makes this job longer, which makes this test less flaky. I didn't find any other solution for now

Reply:
Ok, thx for the clarification. Sounds good then.